### PR TITLE
Added tests for FindStreetcodesWithMatchTitle() FindSortedStreetcodes() FindFilteredStreetcodes() in GetAllStreetcodehandler

### PR DESCRIPTION
**Pull request by maximu211**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: Streetcode/Streetcode.XUnitTest/MediatRTests/StreetCode/Streetcode/GetAllStreetcodesHandlerTests.cs [hunk 1]
```diff
@@ -64,9 +71,121 @@ public async Task Handle_ReturnsCorrectType()
...
+                Sort = "-Title" 
+            };
+
+            var query = new GetAllStreetcodesQuery(request);
+
+            var repositoryWrapperMock = new Mock<IRepositoryWrapper>();
+            var streetcodeRepositoryMock = new Mock<IStreetcodeRepository>();
+            var mapperMock = new Mock<IMapper>();
+
+            repositoryWrapperMock.Setup(x => x.StreetcodeRepository).Returns(streetcodeRepositoryMock.Object);
```
Comment (NadiaKishchuk) [inline Streetcode/Streetcode.XUnitTest/MediatRTests/StreetCode/Streetcode/GetAllStreetcodesHandlerTests.cs:130]: You can create separate method for mocking and setuping objects

**Review by DanyilTerentiev — CHANGES_REQUESTED** [review]
Make separate methods and I approve it

File: Streetcode/Streetcode.XUnitTest/MediatRTests/StreetCode/Streetcode/GetAllStreetcodesHandlerTests.cs [hunk 2]
```diff
@@ -64,9 +71,121 @@ public async Task Handle_ReturnsCorrectType()
...
+            var handler = new GetAllStreetcodesHandler(repositoryWrapperMock.Object, mapperMock.Object);
+
+            IQueryable<StreetcodeContent> streetcodes = new List<StreetcodeContent>().AsQueryable(); 
+
+            // Act
+
+            var result = await handler.Handle(query, default);
+
+            // Assert
+
```
Comment (Tysyatsky) [inline Streetcode/Streetcode.XUnitTest/MediatRTests/StreetCode/Streetcode/GetAllStreetcodesHandlerTests.cs:105]: spacing

**Review by Tysyatsky — COMMENTED** [review]
fix all comments and you are good to go

**Comment by maximu211** [comment]
i fixed it all) check it out

File: Streetcode/Streetcode.XUnitTest/MediatRTests/StreetCode/Streetcode/GetAllStreetcodesHandlerTests.cs [hunk 3]
```diff
@@ -64,9 +71,104 @@ public async Task Handle_ReturnsCorrectType()
             // act
 
             var result = await handler.Handle(new GetAllStreetcodesQuery(request), CancellationToken.None);
-            
+
             // assert
             Assert.IsAssignableFrom<GetAllStreetcodesResponseDTO>(result.Value);
         }
+        [Fact]
```
Comment (OlehHatsko) [inline Streetcode/Streetcode.XUnitTest/MediatRTests/StreetCode/Streetcode/GetAllStreetcodesHandlerTests.cs:78]: Use common formatting conventions (Add empty line between methods)

File: Streetcode/Streetcode.XUnitTest/MediatRTests/StreetCode/Streetcode/GetAllStreetcodesHandlerTests.cs [hunk 4]
```diff
@@ -64,9 +71,104 @@ public async Task Handle_ReturnsCorrectType()
...
+
+            // Act
+            var result = await handler.Handle(query, default);
+
+            // Assert
+            Assert.NotNull(result);
+            Assert.True(result.IsSuccess);
+            Assert.NotNull(result.Value);
+
+            var response = result.Value;
```
Comment (OlehHatsko) [inline Streetcode/Streetcode.XUnitTest/MediatRTests/StreetCode/Streetcode/GetAllStreetcodesHandlerTests.cs:101]: You do not need this variable. You can use result.Value.Streetcodes like parameter

File: Streetcode/Streetcode.XUnitTest/MediatRTests/StreetCode/Streetcode/GetAllStreetcodesHandlerTests.cs [hunk 5]
```diff
@@ -64,9 +71,104 @@ public async Task Handle_ReturnsCorrectType()
...
+            // Arrange
+            var request = new GetAllStreetcodesRequestDTO
+            {
+                Filter = "filter:Filter"
+            };
+
+            var query = new GetAllStreetcodesQuery(request);
+
+            var (handler, repositoryWrapperMock) = SetupMockObjectsAndGetHandler(out var streetcodeRepositoryMock, out var mapperMock);
+            IQueryable<StreetcodeContent> streetcodes = new List<StreetcodeContent>().AsQueryable();
```
Comment (OlehHatsko) [inline Streetcode/Streetcode.XUnitTest/MediatRTests/StreetCode/Streetcode/GetAllStreetcodesHandlerTests.cs:151]: why do you need explicit typing in here? consider common approach for creating variables

**Review by OlehHatsko — APPROVED** [review]
fix comments

**Review by MaksBrat — APPROVED** [review]

**Pull request merged**